### PR TITLE
Update Main.java

### PR DESCRIPTION
**Pull request by Fry07**

### CODE REVIEW

**Comment by Fry07** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [Main.java#L38](https://github.com/Fry07/saas-template/blob/8fc103fbb38f29cd62ea76a46be20d6309ae37d7/services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java#L38): 'password' detected in this expression, review this potentially hardcoded credential. [rule](http://159.89.108.85:9000/coding_rules#rule_key=squid%3AS2068)
1. ![MINOR][MINOR] [Main.java#L37](https://github.com/Fry07/saas-template/blob/8fc103fbb38f29cd62ea76a46be20d6309ae37d7/services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java#L37): Remove this unused "uname" local variable. [rule](http://159.89.108.85:9000/coding_rules#rule_key=squid%3AS1481)
1. ![MINOR][MINOR] [Main.java#L38](https://github.com/Fry07/saas-template/blob/8fc103fbb38f29cd62ea76a46be20d6309ae37d7/services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java#L38): Remove this unused "password" local variable. [rule](http://159.89.108.85:9000/coding_rules#rule_key=squid%3AS1481)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by dpetrovs — APPROVED** [review]
Ok

File: services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java [hunk 1]
```diff
@@ -27,15 +27,15 @@
...
 
-  public static Integer i = new Integer(123);
+  //public static Integer i = new Integer(123);
   
   static {
     PROTOCOL = "http://";
     host = Optional.ofNullable(System.getenv("SERVICES_GJ_HOSTNAME"));
     port = Optional.ofNullable(System.getenv("SERVICES_GJ_PORT"));
     BASE_URI = PROTOCOL + host.orElse("localhost") + ":" + port.orElse("80") + "/";
     String uname = "steve";
```
Comment (Fry07) [inline services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java:37]: MINOR Remove this unused "uname" local variable. [rule](http://159.89.108.85:9000/coding_rules#rule_key=squid%3AS1481)

File: services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java [hunk 2]
```diff
@@ -27,15 +27,15 @@
...
-    String password = "blue";
+    //String password = "blue";
```
Comment (Fry07) [inline services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java:38]: MAJOR This block of commented-out lines of code should be removed. [rule](http://159.89.108.85:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by Fry07** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by dpetrovs — APPROVED** [review]
Ok2

**Review by dpetrovs — APPROVED** [review]

**Pull request merged**